docs: workflow status on dashboard (#190)
chaos-mesh/website: docs/check-workflow-status.md

title: Check Workflow Status
---

> ## Check workflow status using Chaos Dashboard

Reviewer: ```suggestion
## Check Workflow status using Chaos Dashboard
```

> 1. List all the workflows in the Chaos Dashboard.
> 
> ![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)
> 
> 2. Select the workflow you want to check, show the details of the workflow.
> 
> ![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)

Reviewer: ```suggestion
1. List all Workflow in Chaos Dashboard.

![List Workflow On Dashboard](./img/list-workflow-on-dashboard.png)

2. Select the Workflow you want to check the status, see the details of the Workflow.

![Workflow Status On Dashboard](./img/workflow-status-on-dashboard.png)
```

> ## Check workflow status using `kubectl` commands

Reviewer: ```suggestion
## Check Workflow status using `kubectl` commands
```
